public/service_portal/v0.1.0/openapi.json

```diff
@@ -818,6 +895,15 @@
                     "type": "string"
                 }
             },
+            "IconIDs": {
+                "name": "icon_ids",
```

Comment:
@lindgrenj6 Is this an array of ids? or a single id?

Reply:
I is an array of ids, separated by commas. As we discussed earlier today so that the frontend can request as many icons as they want.  

Requests can be made like so:
`/icons/1` 
or 
`/icons/1,2,3,4`

Reply:
@lindgrenj6 Shouldn't that come in the body or as URL encoded array attribute I am not sure if you need to get bulk data back if you string together ids with commas. Have you seen examples  of this for a REST API.

Reply:
I've seen it done in the past - I guess I didn't know that wasn't an ok way to do it. Would you rather me use a path param like this:  `/icons?bulk_ids=1,2,3,4` or add a POST endpoint to `/icons` that has a body of the id's requested for bulk

Reply:
The `/1,2,3...` pattern is being used in other insights APIs already (in inventory API for instance). I personally don't care that much about the format but consistency across multiple services would be nice imo.